core/src/Authorization.html

```diff
@@ -194,9 +194,9 @@
             checkAuth();
           })
           .catch(err => {
-            console.error(
-              `Could not instantate ${idpProviderName} provider : ${err}`
-            );
+            const errorMsg = `Error: ${err.message || err}`;
+            console.error(errorMsg, err);
```

Comment:
I think this can be just `console.error(errorMsg);`

Reply:
This part is inside an if block:
```
      if (GenericHelpers.isPromise(idpProviderInstance)) {
```
Looks like we will not display the error if the `idpProviderInstance` is not a promise.

Reply:
I updated the implementation to output the second err (object) only in case there is a message found.
If the Provider does not return a Promise, it is up to the provider itself to throw any required error.